charts/incubator/awesome-ttrss/Chart.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: v2
+appVersion: "latest-2022-04-19"
+description: Awesome TTRSS aims to provide a powerful Dockerized all-in-one solution for Tiny Tiny RSS
+name: awesome-ttrss
+version: 0.0.1
+kubeVersion: ">=1.16.0-0"
+keywords:
+- tt-rss
+- awesome-ttrss
+home: https://github.com/truecharts/apps/tree/master/charts/stable/awesome-ttrss
+icon: https://truecharts.org/_static/img/appicons/tt-rss.png
```

```suggestion
icon: https://truecharts.org/_static/img/appicons/awesome-ttrss.png
```

let's keep the names consistent :)

---

The icon of this application can be used directly from ttrss, its own icon is an emoji